services/src/test/java/edu/unc/lib/dl/cdr/services/rest/AccessControlRetrievalIT.java

```diff
@@ -97,14 +91,10 @@ public void init_() throws Exception {
                 "text.txt", "text/plain", null, null);
 
         workObj.setPrimaryObject(fileObj.getPid());
-        workObj.addDescription(getClass().getResourceAsStream("/datastreams/simpleMods.xml"));
-        uuid = workObj.getPid().getUUID();
-
-        objPrincRoles = new HashMap<>();
-        addPrincipalRoles(objPrincRoles, "admin", unitOwner);
-        addPrincipalRoles(objPrincRoles, AUTHENTICATED_PRINC, canAccess);
+        uuid = workObj.getPid().getId();
     }
 
+    @SuppressWarnings("unchecked")
     @Test
     public void testGetPermissionsObjectWithChildren() throws Exception {
```

Comment:
This test should verify that the children objects were returned in the response too